src/middlewared/middlewared/alert/source/cores.py

```diff
@@ -0,0 +1,22 @@
+import os
+
+from middlewared.alert.base import AlertClass, AlertCategory, AlertLevel, Alert, ThreadedAlertSource
+
+
+class CoreFilesArePresentAlertClass(AlertClass):
+    category = AlertCategory.SYSTEM
+    level = AlertLevel.WARNING
+    title = "Core Files Are Present"
+    text = ("Core files are present in /var/db/system/cores: %s. Please, file a ticket at https://jira.ixsystems.com/ "
```

Comment:
Not sure if there is any usefulness in listing every file here.

Reply:
> I wonder if we should compare the creation date of these files and only account for relatively newer ones... e.g. within last 5 days?

I don't think that'll be right, files older than 5 days are still consuming disk space

Reply:
We could delete older files?
Not sure its useful to keep old files. i have seen reports back fro.2018